public/app/features/playlist/PlaylistEditPage.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react';
+import { connect, MapStateToProps } from 'react-redux';
+import { NavModel } from '@grafana/data';
+import { locationService } from '@grafana/runtime';
+import { useStyles } from '@grafana/ui';
+
+import Page from 'app/core/components/Page/Page';
+import { StoreState } from 'app/types';
+import { GrafanaRouteComponentProps } from '../../core/navigation/types';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { PlaylistForm } from './PlaylistForm';
+import { updatePlaylist } from './api';
+import { Playlist } from './types';
+import { usePlaylist } from './usePlaylist';
+import { getPlaylistStyles } from './styles';
+
+interface ConnectedProps {
+  navModel: NavModel;
+}
+
+export interface RouteParams {
+  id: number;
+}
+
+interface Props extends ConnectedProps, GrafanaRouteComponentProps<RouteParams> {}
+
+export const PlaylistEditPage: FC<Props> = ({ navModel, match }) => {
+  const styles = useStyles(getPlaylistStyles);
+  const { playlist, loading } = usePlaylist(match.params.id);
+  const onSubmit = async (playlist: Playlist) => {
+    await updatePlaylist(match.params.id, playlist);
+    locationService.push('/playlists');
+  };
+
+  return (
+    <Page navModel={navModel}>
+      <Page.Contents isLoading={loading}>
+        <div className="page-container page-body">
```

Comment:
The Page.Contents component adds these classes already so this adds some double page paddings :)

Reply:
Great catch, I was thinking about migrating those classes to emotion but I don't need to know 👍